acceptance/testdata/release/release-upload-download.txtar

```diff
@@ -0,0 +1,26 @@
+# Create a repository with a file so it has a default branch
+exec gh repo create $ORG/$SCRIPT_NAME-$RANDOM_STRING --add-readme --private
+
+# Defer repo cleanup
+defer gh repo delete --yes $ORG/$SCRIPT_NAME-$RANDOM_STRING
+
+# Clone the repo
+exec gh repo clone $ORG/$SCRIPT_NAME-$RANDOM_STRING
+
+# Create a release in the repo
+cd $SCRIPT_NAME-$RANDOM_STRING
+exec gh release create v1.2.3 --notes 'awesome release' --latest
+
+# Upload an asset to the release
+exec gh release upload v1.2.3 ../asset.txt
```

Comment:
This currently fails when targeting ghe.com. It is being tracked by the owning API team.